### PR TITLE
Add Cisco C9200L-24PXG-2Y

### PR DESCRIPTION
**Pull request by okoelbl**
Add Cisco C9200L-24PXG-2Y switch

### CODE REVIEW

File: device-types/Cisco/C9200L-24PXG-2Y.yaml [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+    type: rj-45
+  - name: usb
+    type: usb-mini-b
+power-ports:
+  - name: PS-1
+    type: iec-60320-c14
+    maximum_draw: 600
+  - name: PS-2
+    type: iec-60320-c14
+    maximum_draw: 600
```
Comment (danner26) [inline device-types/Cisco/C9200L-24PXG-2Y.yaml:20]: These PSU's are modular and thus you should define `module-bays` with 2 slots. Then, the applicable PSU modules should be added to the module-types folder if they do not already exist

File: device-types/Cisco/C9200L-24PXG-2Y.yaml [hunk 2]
```diff
@@ -0,0 +1,128 @@
+---
+manufacturer: Cisco
+model: C9200L-24PXG-2Y
+part_number: C9200L-24PXG-2Y
+slug: cisco-c9200l-24pxg-2y
+u_height: 1
+is_full_depth: false
```
Comment (danner26) [inline device-types/Cisco/C9200L-24PXG-2Y.yaml:7]: Hello, thank you for your contribution! In an attempt to have the most accurate and complete data we can, I have a few additional data points we would like you to add if possible.

1. Please add the `airflow` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not
2. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

If you have any questions, please let us know :)

**Review by danner26 — CHANGES_REQUESTED** [review]

File: device-types/Cisco/C9200L-24PXG-2Y.yaml (hunk 2 above)
Comment (okoelbl) [reply device-types/Cisco/C9200L-24PXG-2Y.yaml:10]: Hello and thank you for reviewing! I've added the requested data points and module bays, as well as the supported (AC) PSU.

File: device-types/Cisco/C9200L-24PXG-2Y.yaml (hunk 1 above)
Comment (okoelbl) [reply device-types/Cisco/C9200L-24PXG-2Y.yaml]: Thanks! Added module bays (see other reply)

**Review by danner26 — APPROVED** [review]

**Pull request merged**